Library/ViewModels/SetYourPasswordViewModel.swift

```diff
@@ -1,10 +1,10 @@
 import Foundation
 import KsApi
+import Prelude
```

Comment:
Is there an instance where this is used?

Reply:
I'm using `IsTrue` on line: 69...

` let saveAction = formIsValid
.takeWhen(submitFormEvent)
      .filter(isTrue)
      .ignoreValues()`